PayPal/src/main/java/com/braintreepayments/api/paypal/AmountBreakdown.kt

```diff
@@ -0,0 +1,44 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * A recurring billing amount breakdown.
+ *
+ * This object is only used in `PayPalCheckoutRequest` to customize how the transaction amount is
```

Comment:
One more doc string nitpick - can we update everywhere we added backticks to use brackets [ ] instead? this allows actual parameter/class linking in Kdoc